src/components/checkbox/checkbox.scss

```diff
@@ -3,7 +3,7 @@
 @import "variables";
 
 /** The width/height of the checkbox element. */
-$md-checkbox-size: 18px !default;
+$md-checkbox-size: 20px !default;
```

Comment:
Can we refactor this out into something like `$md-toggle-size` and use it for both checkbox and radio button?